cdap-examples/WikipediaPipeline/src/main/java/co/cask/cdap/examples/wikipedia/package-info.java

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
No, you're right. I'm wrong. You can indent all the following lines if you want everything lined up...